source/frontend/src/features/research/list/ResearchFilter/ResearchFilter.tsx

```diff
@@ -72,51 +72,51 @@ export const ResearchFilter: React.FunctionComponent<
       {formikProps => (
         <FilterBoxForm className="p-3">
           <Row>
-            <Col xl="1">
+            <Col lg="1">
```

Comment:
The research filter displays fine at 992px - no need to go one row per line unless below 992.